### PR TITLE
Simple nlog docs

### PR DESCRIPTION
**Pull request by chrisdunelm**
Updates #2185

### CODE REVIEW

File: apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs [hunk 1]
```diff
@@ -40,56 +40,69 @@ public void Overview()
...
                 {
-                    xmlString = reader.ReadToEnd();
+                    xml = reader.ReadToEnd();
                 }
             }
-            xmlString = xmlString.Replace("PROJECT_ID", projectId).Replace("LOG_ID", logId);
-            var xmlReader = XmlReader.Create(new MemoryStream(Encoding.UTF8.GetBytes(xmlString)));
+            xml = xml.Replace("PROJECT_ID", projectId).Replace("LOG_ID", logId);
+            Assert.False(File.Exists(fileName), "Test would overwrite existing file");
+            //var xmlReader = XmlReader.Create(new MemoryStream(Encoding.UTF8.GetBytes(xmlString)));
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs]: Remove the commented out line?

File: apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs [hunk 2]
```diff
@@ -40,56 +40,69 @@ public void Overview()
...
+            File.WriteAllText(fileName, xml);
 
             // Ensure NLog throws exceptions for any error conditions
             LogManager.ThrowExceptions = true;
             LogManager.ThrowConfigExceptions = true;
 
             // Configure nlog to use Google Stackdriver logging from the XML configuration file.
-            LogManager.Configuration = new XmlLoggingConfiguration(xmlReader, "unused");
+            //LogManager.Configuration = new XmlLoggingConfiguration(xmlReader, "unused");
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs]: And again?

File: apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs [hunk 3]
```diff
@@ -40,56 +40,69 @@ public void Overview()
...
+                // Configure nlog to use Google Stackdriver logging from the XML configuration file.
+                LogManager.LoadConfiguration("nlog.xml");
 
-            // Flush buffered log entries before program exit; then shutdown the logger before program exit.
-            LogManager.Flush(TimeSpan.FromSeconds(15));
-            LogManager.Shutdown();
-            // End sample
+                // Acquire a logger for this class
+                Logger logger = LogManager.GetCurrentClassLogger();
+                // Log some information.This log entry will be sent to Google Stackdriver Logging.
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs]: Nit: space after period.

File: apis/Google.Cloud.Logging.NLog/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{{title}}
+
+`Google.Cloud.Logging.NLog` is a .NET client library to integrate  [Google Stackdriver
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.NLog/docs/index.md]: Nit: double space

File: apis/Google.Cloud.Logging.NLog/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+Logging](https://cloud.google.com/logging/) with
+[NLog](http://nlog-project.org/).
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.NLog/docs/index.md]: Unwrap onto the previous line? It's pretty short.

File: apis/Google.Cloud.Logging.NLog/docs/index.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+{{installation}}
+
+Install the `Google.Cloud.Logging.NLog` package from NuGet. Add it to
```
Comment (jskeet) [inline apis/Google.Cloud.Logging.NLog/docs/index.md]: Remove this paragraph - it's generated by the `{{installation}}` part.

File: apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs (hunk 1 above)
Comment (chrisdunelm) [reply apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs]: Oops, yes, trying to do things too fast!

File: apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs (hunk 2 above)
Comment (chrisdunelm) [reply apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs]: done

File: apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs (hunk 3 above)
Comment (chrisdunelm) [reply apis/Google.Cloud.Logging.NLog/Google.Cloud.Logging.NLog.Snippets/GoogleStackdriverTargetSnippets.cs]: done

File: apis/Google.Cloud.Logging.NLog/docs/index.md (hunk 4 above)
Comment (chrisdunelm) [reply apis/Google.Cloud.Logging.NLog/docs/index.md]: done

File: apis/Google.Cloud.Logging.NLog/docs/index.md (hunk 5 above)
Comment (chrisdunelm) [reply apis/Google.Cloud.Logging.NLog/docs/index.md]: done

File: apis/Google.Cloud.Logging.NLog/docs/index.md (hunk 6 above)
Comment (chrisdunelm) [reply apis/Google.Cloud.Logging.NLog/docs/index.md]: Done. Also removed from log4net

**Pull request merged**